iot-hub/Samples/service/CleanupDevicesSample/ImportExportDevicesHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    internal static class ImportExportDevicesHelpers
+    {
+        private static readonly char[] s_newlines = new char[]
+        {
+            '\r',
+            '\n',
+        };
+
+        /// <summary>
+        /// Makes a stream compatible for writing to a storage blob of serialized, newline-delimited rows of the specified devices
+        /// </summary>
+        /// <param name="devices">The devices to serialize</param>
+        internal static Stream BuildDevicesStream(IReadOnlyList<ExportImportDevice> devices)
+        {
+            var devicesFileSb = new StringBuilder();
+
+            foreach (ExportImportDevice device in devices)
+            {
+                devicesFileSb.AppendLine(JsonConvert.SerializeObject(device));
+            }
+
+            byte[] devicesFileInBytes = Encoding.Default.GetBytes(devicesFileSb.ToString());
+            return new MemoryStream(devicesFileInBytes);

```

Comment:
`MemoryStream` is IDisposable. I can't find the code that calls this method, but I wanted to make sure you checked that it was being disposed by the caller.

Reply:
It wasn't being disposed, I'll fix it.